oct/frontend/src/routes/login/page.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+import Cookies from "universal-cookie";
+
+const cookies = new Cookies();
+
+export default function Login() {
+  const location = useLocation();
+  const queryParams = new URLSearchParams(location.search);
+
+  function isResponseOk(response: Response) {
+    if (response.status >= 200 && response.status <= 299) {
+      return response.text().then((text) => {
+        return JSON.parse(text);
+      });
+    } else {
+      throw Error(response.statusText);
+    }
+  }
+
+  useEffect(() => {
+    fetch("/api/login/", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "X-CSRFToken": cookies.get("csrftoken"),
```

Comment:
csrftoken should already get passed with cookies

Reply:
doesn't seem like it does, when i remove this i get a 403 error on the django side

Reply:
nvm this approach is fine